tests/devapps/NetCoreTestApp/Program.cs

```diff
@@ -275,6 +280,26 @@ 1. IWA
                             }
 
                             break;
+
+                        case 12: // acquire token interactive with WamBroker
+                            {
+                                var optionsbroker = new SystemWebViewOptions()
+                                {
+                                    //BrowserRedirectSuccess = new Uri("https://www.bing.com?q=why+is+42+the+meaning+of+life")
```

Comment:
?

Reply:
Copy paste from #238. https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/f0ec8124f6d927bea3a3f1040197e26c74066a90/tests/devapps/NetCoreTestApp/Program.cs#L238
Will remove from both the places.